packages/design-system/src/DatePicker/styles.css

```diff
@@ -217,4 +218,29 @@
 
 .react-datepicker__children-container {
   margin-top: -10px !important;
```

Comment:
Anyone knows why we have this style written?
`margin-top: -10px !important;`
cc: @tanvibhakta @rohitagarwal88

Reply:
I think it's because the spacing of the child container otherwise was too large and didn't fit the design spec.